src/cli/utils.js

```diff
@@ -52,11 +53,9 @@ exports.getIPFS = (argv, callback) => {
     pass: argv.pass
   })
 
-  const cleanup = promisify((cb) => {
+  const cleanup = callbackify(async () => {
     if (node && node._repo && !node._repo.closed) {
-      node._repo.close((err) => cb(err))
-    } else {
-      cb()
+      await node._repo.close()
     }
   })
```

Comment:
ignore this !! promise-nodeify would be better but lets get this merged

Reply:
TBH I reached for `callbackify` because it had a nicely symmetrical name to `promisify`.  I have no strong opinion on which is better so happy to swap it out in a future PR.